sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -282,6 +284,13 @@ def time_to_live(self, value):
         else:
             self._amqp_header.time_to_live = int(value) * 1000
 
+        if self._amqp_header.time_to_live and self._amqp_header.time_to_live != MAX_DURATION_VALUE:
```

Comment:
does JS and .NET default `time_to_live` to MAX_DURATION_VALUE when constructing the message?

Reply:
yep! --> https://github.com/Azure/azure-sdk-for-js/blob/f53488f738ceb7c76fd1b76f67f36e1c8c868fc0/sdk/servicebus/service-bus/src/serviceBusMessage.ts#L323

Reply:
okay, I was wondering if JS/.NET would internally update amqp header time_to_live to be the MAX_DURATION_VALUE for outgoing messages, but they seems not